jobs/github-event/push.js

```diff
@@ -57,7 +57,20 @@ module.exports = async function (data) {
   // always put package.jsons in the repoDoc (new & old)
   // if remove event: delete key of package.json
   const oldPkg = _.get(repoDoc, ['packages'])
-  await updateRepoDoc({installationId: installation.id, doc: repoDoc, log})
+  try {
+    await updateRepoDoc({installationId: installation.id, doc: repoDoc, log})
+  } catch (e) {
+    console.log(e)
```

Comment:
Should this be a `log` log instead of a `console.log` log? Also, is this the only possible reason `updateRepoDoc` could fail?

Reply:
log: yes 😇 

only reason: possibly, I’ll have a look